trace-forward/README.md

```diff
@@ -0,0 +1,5 @@
+# trace-forward
+
+`trace-forward` is a library allowing to forward tracing items from one process to another one. It is built upon [`typed-protocols`](https://github.com/input-output-hk/ouroboros-network/tree/master/typed-protocols).
+
+The `trace-dispatcher` is using `trace-forward` to forward `TraceObject`s from the node to exernal acceptors (for example, `cardano-tracer`).
```

Comment:
Can we add the devs who are responsible for this library here?

Reply:
Yes, we can. But should I add them here? Or it should be added in `CODEOWNERS` file?

Reply:
Done.

Reply:
You should add them here.

Reply:
Ok.